socorro/processor/breakpad_transform_rules.py

```diff
@@ -73,50 +74,42 @@ class ExternalProcessRule(Rule):
     )
     required_config.add_option(
         'command_line',
-        doc=(
-            'the template for the command to invoke the external program; uses Python format '
-            'syntax'
-        ),
-        default=(
-            'timeout -s KILL 30 {command_pathname} 2>/dev/null'
-        ),
+        doc='template for the command to invoke the external program; uses Python format syntax',
+        default='timeout -s KILL 30 {command_pathname} 2>/dev/null',
     )
     required_config.add_option(
         'command_pathname',
-        doc='the full pathname to the external program to run '
-        '(quote path with embedded spaces)',
+        doc='the full pathname to the external program to run (quote path with embedded spaces)',
         default='/data/socorro/stackwalk/bin/dumplookup',
     )
     required_config.add_option(
         'result_key',
-        doc='the key where the external process result should be stored '
-            'in the processed crash',
-        default='%s_result' %
-            required_config.command_pathname.default.split('/')[-1]
-            .replace('-', ''),
+        doc='where the external process result should be stored in the processed crash',
+        default='%s_result' % (
+            required_config.command_pathname.default.split('/')[-1].replace('-', '')
+        ),
     )
     required_config.add_option(
         'return_code_key',
-        doc='the key where the external process return code should be stored '
-            'in the processed crash',
-        default='%s_return_code' %
-            required_config.command_pathname.default.split('/')[-1],
+        doc='where the external process return code should be stored in the processed crash',
+        default='%s_return_code' % required_config.command_pathname.default.split('/')[-1],
     )
 
     def __init__(self, config):
         super(ExternalProcessRule, self).__init__(config)
 
     def _interpret_external_command_output(self, fp, processor_meta):
+        data = fp.read()
         try:
-            return ujson.load(fp)
+            return json.loads(data)
         except Exception as x:
+            self.config.logger.error(
+                '%s non-json output: "%s"' % (self.config.command_pathname, data[:100])
+            )
```

Comment:
I dropped ujson here and also moved the `fp.read()` to a separate line. That lets us spit out some of it to the logs if it throws an error in `json.loads` which makes it muuuuuch more likely to be seen when doing development.